sentences/pl/_common.yaml

```diff
@@ -76,38 +94,248 @@ lists:
         out: "closing"
   cover_classes:
     values:
-      - in: (drzwi garażowe|drzwi w garażu)
-        out: garage
-      - in: bram(a|y)
-        out: gate
-      - in: zasłon(a|y)
-        out: curtain
-      - in: markiz(a|y)
-        out: awning
-      - in: rolet(a|y)
-        out: blind
-      - in: rolet(a|y)
-        out: shade
-      - in: żaluzj(a|e)
-        out: shutter
-      # - in: okn(o|a)
-      #   out: window
+      - in: "markiz[a|y]"
+        out: "awning"
+      - in: "rolet[a|y]"
+        out: "blind"
+      - in: "zasłon[a|y]"
+        out: "curtain"
+      - in: "drzwi"
+        out: "door"
+      - in: "(garaż(e)|brama garażowa|(drzwi|bramy) garażowe|(drzwi|brama|bramy) do garażu)"
+        out: "garage"
+      - in: "bram[a|y]"
+        out: "gate"
+      - in: "osłon[a|y]"
+        out: "shade"
+      - in: "żaluzj(i|a|e)"
+        out: "shutter"
+      - in: "ok(ien|no|na)"
+        out: "window"
+      - in: "klap[a|y]"
+        out: "damper"
   lock_states:
     values:
       - in: "zablokowan(e|y|ych)|zamknięt(y|e|ych)"
         out: "locked"
       - in: "odblokowan(e|y|ych)|otwart(e|y|ych)"
         out: "unlocked"
 
+  # binary_sensor
+  bs_battery_states:
+    values:
+      - in: "niski"
+        out: "on"
+      - in: "normalny"
+        out: "off"
+
+  bs_battery_charging_states:
+    values:
+      - in: "ładuje się"
+        out: "on"
+      - in: "nie ładuje się"
+        out: "off"
+
+  bs_carbon_monoxide_states:
+    values:
+      - in: "(wykryty|wyzwolony|włączony)"
+        out: "on"
+      - in: "brak"
+        out: "off"
+
+  bs_cold_states:
+    values:
+      - in: "zimno"
+        out: "on"
+      - in: "normalnie"
+        out: "off"
+
+  bs_connectivity_states:
+    values:
+      - in: "połaczony"
```

Comment:
missing `ą`

Reply:
Most of the `sensor` and `binary_sensor` entities are not used yet. I'll just prepare the configuration for them. It will be quite complicated so I didn't want to combine it with this configuration.